nvflare/app_common/widgets/intime_model_selector.py

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2022, NVIDIA CORPORATION.  All rights reserved.
+# Copyright (c) 2022-2023, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
From what I searched and other open source framework's code, I think we could just put create year here.
Otherwise we need to keep updating all the headers of all files we touch in PRs.

Reply:
Good point. I reverted to 2022.